### DISCUSSION TITLE
[회의] 22-03-23-수

### DISCUSSION THREAD

**Discussion by gomarag** [codestates/siphoon-note#69] [회의록]
### 질문
- [x] 푸터 컴포넌트에는 어떤 내용이 들어가게 되나요?👀

### 조율
- [x] 일기 수정 기능을 더할 수 있을지

### 제안
- [x] ~현재 Projects 탭에서 사용중인 TODO를 주차별로 관리하면 어떨지 -> 컬럼이 많아지면서 오늘 컬럼이 옆으로 밀려서 잘 안 보임~
- [ ] 오후3시 미팅 전에 팀원들이 푸시한 `코드`나, `투두`, `문서`를 훑어보고 생긴 질문이나, 조율할 부분들은 `여기 회의록`에 서로 적어두고 회의 진행하면 좋을 것 같음
---
## 📌정리
1. 일기 카드는 일기 내용을 `잘라서` 보여줌 -> 서버도 일기 내용은 자르고, 일기 개수는 10~15개씩 조회.
2. 개별 카드 클릭 시 모달 창에서 전체 일기를 보여주면서 `수정` 버튼이 보임.

**Comment by Leekyeongmi** [comment]
앗, 저 안그래도 일기 수정하는 기능에 대해서 이야기 해보고 싶었어요! 일기 더보기를 통해서 글 전체를 볼 수 있다면, 수정하는 기능도 추가하면 좋을 거 같아요 ㅎㅎ..
